### PR TITLE
BossBar helper

### PR DESCRIPTION
**Pull request by MrScopes**
this pr adds `bossbar` to velt/helpers

### CODE REVIEW

File: src/main/resources/@types/velt/helpers.d.ts [hunk 1]
```diff
@@ -7,601 +7,600 @@
...
+    setProgress(progress: number): this;
+    /** Set the bossbar's title. */
+    setTitle(title: string): this;
+    /** Set the bossbar's color. */
+    setColor(color: BarColor): this;
+    /** Set the bossbar style. */
+    setStyle(style: BarStyle): this;
+    /** The bossbar's color. */
+    color: string;
+    /** The player's the bossbar is attached to. */
```
Comment (Grabsky) [inline src/main/resources/@types/velt/helpers.d.ts]: shouldn't this be `players`?
```suggestion
    /** The players the bossbar is attached to. */
```

**Review by Grabsky — CHANGES_REQUESTED** [review]

**Review by Cormanz — COMMENTED** [review]
Rename namespacekey method to namespaceKey
Make getBossBar, getBossBars, and createBossBar also part of helpers

**Comment by MrScopes** [comment]
> Rename namespacekey method to namespaceKey
> Make getBossBar, getBossBars, and createBossBar also part of helpers

it's already a part of helpers?

**Comment by Cormanz** [comment]
I mean helpers directly.

**Comment by MrScopes** [comment]
> I mean helpers directly.

as in remove the crafting interface?

**Comment by Cormanz** [comment]
Nah, I just mean add the bossbar methods directly to helpers as well as an alias.

**Comment by Cormanz** [comment]
Alternatively bossbar.create and bossbar.get would be better since it doesn't mean you have to repeat the bossbar name twice

**Review by Cormanz — APPROVED** [review]

**Pull request merged**